src/components/EditSections/EditServicePoints/EditServicePoints.js

```diff
@@ -174,14 +174,10 @@ class EditServicePoints extends React.Component {
             onToggle={this.props.onToggle}
             displayWhenClosed={<Badge>{(this.userServicePoints && this.userServicePoints.length) || 0}</Badge>}
           >
-            <Row>
-              <Col xs={8}>
-                { this.renderAddServicePointButton() }
-                { this.renderPreferredServicePointSelect() }
-                { this.renderServicePoints() }
-                { this.renderAddServicePointModal() }
-              </Col>
-            </Row>
+            <div>{ this.renderAddServicePointButton() }</div>
```

Comment:
Before, this whole section was had a full-width wrapper. I'm assuming these other methods render their own wrapper, or they don't need it at all?